include/LightGBM/utils/text_reader.h

```diff
@@ -84,6 +84,17 @@ class TextReader {
   * \return Text data, store in std::vector by line
   */
   inline std::vector<std::string>& Lines() { return lines_; }
+  /*!
+  * \brief Get joined text data that read from file
+  * \return Text data, store in std::string, joined all lines by delimiter "\n"
+  */
+  inline std::string JoinedLines() {
```

Comment:
I believe that delimiter can be an argument of this function given the general purpose of this class.

Reply:
Thanks for your suggestions. Let's do this in a follow-up PR. Since this one is emergency for our product.

Reply:
Done.